stable/prometheus/templates/server-clusterrole.yaml

```diff
@@ -32,7 +32,7 @@ rules:
       - list
       - watch
   - apiGroups:
-      - "extensions"
```

Comment:
maybe this should be an addition instead of replacement in order to support newer and older clusters?

Reply:
Good )